### PR TITLE
Add victoriametrics to landscape model

### PR DESCRIPTION
**Pull request by valyala**
Updates https://github.com/openapm/landscape-model/issues/106

### CODE REVIEW

File: model/components/victoriametrics.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      - VictoriaMetrics/VictoriaMetrics/tree/master/app/vmalert
+    twitter: MetricsVictoria
+    logo: victoriametrics
+    license:
+      - Apache License 2.0
+    categories:
+      - alerting
+      - data-processing
+    connections:
+      dataFrom:
```
Comment (AlexanderWert) [inline model/components/victoriametrics.yml:65]: According to the description of this component, vmalert is compatible with components that support Prometheus Querying API. So I would assume that it is at least compatible with Victoriametrics itself and Prometheus, correct? Shouldn't it then appear in this list as well. Which would mean that vmalert can alert on data from Victoriametrics and Prometheus.

Comment (valyala) [reply model/components/victoriametrics.yml:65]: Correct - vmalert can alert on data from any service that exposes Prometheus querying API. This includes VictoriaMetrics, Prometheus, Cortex and M3DB. Will update the PR accordingly.

Comment (valyala) [reply model/components/victoriametrics.yml:65]: Done

**Comment by AlexanderWert** [comment]
HI @valyala ,

vitoriametrics is now available on openapm.io:

https://openapm.io/landscape?agent=vmagent,prometheus-exporter&instrumentation-lib=opentelemetry,micrometer&storage=victoriametrics&dashboarding=grafana&alerting=grafana,vmalert&data-processing=vmalert&usedges=vmalert:victoriametrics&showCommercial=true&showFormats=false

Regards,
Alex

**Comment by valyala** [comment]
Thanks!

**Pull request merged**